cli/prerender.js

```diff
@@ -0,0 +1,30 @@
+// Load zone.js for the server.
+require('zone.js/dist/zone-node');
+const fs = require('fs');
+
+// Import renderModuleFactory from @angular/platform-server.
+const { renderModuleFactory } = require('@angular/platform-server');
+
+// Import module map for lazy loading
+const { provideModuleMap } = require('@nguniversal/module-map-ngfactory-loader');
+
+const files = fs.readdirSync(`${process.cwd()}/dist/dist-server`);
+const mainFiles = files.filter(file => file.startsWith('main'));
+const hash = mainFiles[0].split('.')[1];
```

Comment:
Kinda crude, but not sure if there's a better way?
None of the CLI's output shows us the Hash generated

Reply:
I think it will work fine since there is always just one production build on the dist directory.

Reply:
it appears that that method is just output formatting. It's webpack that generates the hash from memory

Reply:
I think it's okay to do `--output-hashing=false` so I've adjusted the hash grabbing to work when people use this option